src/helpers/dimensions.ts

```diff
@@ -155,6 +155,56 @@ function getBarDimensions(mdObject: VisualizationObject.IVisualizationObjectCont
     ];
 }
 
+function getAreaDimensions(mdObject: VisualizationObject.IVisualizationObjectContent): AFM.IDimension[] {
+    const view: VisualizationObject.IBucket = mdObject.buckets
+            .find(bucket => bucket.localIdentifier === VIEW);
+
+    const stack: VisualizationObject.IBucket = mdObject.buckets
+        .find(bucket => bucket.localIdentifier === STACK);
```

Comment:
I think we should separate retrieving view and stack items to helper methods to reuse it somewhere.